exist-core/src/main/java/org/exist/xquery/value/DayTimeDurationValue.java

```diff
@@ -230,7 +231,7 @@ public ComputableValue div(ComputableValue other) throws XPathException {
         if (other.getType() == Type.DAY_TIME_DURATION) {
             final DecimalValue a = new DecimalValue(secondsValueSigned());
             final DecimalValue b = new DecimalValue(((DayTimeDurationValue) other).secondsValueSigned());
-            return new DecimalValue(a.value.divide(b.value, 20, BigDecimal.ROUND_HALF_UP));
+            return new DecimalValue(a.value.divide(b.value, 20, RoundingMode.HALF_UP));
```

Comment:
In JDK 8 this isn't deprecated. I am interested... where did you see the deprecation?

Reply:
Hmmm Intellij gave this suggestion....

Reply:
https://docs.oracle.com/javase/7/docs/api/java/math/RoundingMode.html // deprecated since java9 https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/BigDecimal.html#ROUND_UNNECESSARY

Reply:
Ah okay, since Java 9. That explains why I didn't see it in JDK 8 ;-)